doc/releases/changelog-dev.md

```diff
@@ -133,43 +133,20 @@
   [(#6396)](https://github.com/PennyLaneAI/pennylane/pull/6396)
 
 
-<h4>New API for Qubit Mixed</h4>
-
-* Added `qml.devices.qubit_mixed` module for mixed-state qubit device support [(#6379)](https://github.com/PennyLaneAI/pennylane/pull/6379). This module introduces an `apply_operation` helper function that features:
-
-  * Two density matrix contraction methods using `einsum` and `tensordot`
-
-  * Optimized handling of special cases including: Diagonal operators, Identity operators, CX (controlled-X), Multi-controlled X gates, Grover operators
-
-* Added submodule 'initialize_state' featuring a `create_initial_state` function for initializing a density matrix from `qml.StatePrep` operations or `qml.QubitDensityMatrix` operations.
-  [(#6503)](https://github.com/PennyLaneAI/pennylane/pull/6503)
-  
 * Added support for constructing `BoseWord` and `BoseSentence`, similar to `FermiWord` and `FermiSentence`.
   [(#6518)](https://github.com/PennyLaneAI/pennylane/pull/6518)
 
-* Added method `preprocess` to the `QubitMixed` device class to preprocess the quantum circuit before execution. Necessary non-intrusive interfaces changes to class init method were made along the way to the `QubitMixed` device class to support new API feature.
+* New API for `default_mixed`: migrate the old `DefaultMixed` to follow our new device API standard. Basically, previous class `qml.devices.default_mixed.DefaultMixed` was refactored. Users will not experience any differences in principle, but for some users who delved deep into PennyLane codebase and happened to use certain deprecated methods or properties of the legacy code this might introduce breaking changes.
```

Comment:
```suggestion
* The `DefaultMixed` device has been refactored to extend the new device API.
```
I don't think you need to explicitly say that "this is not going to break things", I feel that this is implied, otherwise this entry would be in the "breaking changes" section.